ckanext/scheming/templates/scheming/form_snippets/datetime.html

```diff
@@ -0,0 +1,42 @@
+{% import 'macros/form.html' as form %}
+{% set date = '' %}
+{% set time = '' %}
+{% set dateTime = data.get(field.field_name, '') %}
+
+{% if dateTime is string %}
+    {% set parts = dateTime.split() %}
+    {% set date = parts[0] %}
+    {% set time = parts[1] %}
+{% elif dateTime is sequence %}
+    {% set date = dateTime[0] %}
+    {% set time = dateTime[1] %}
```

Comment:
this if/elif is unnecessary. The form snippet will always receive the converted string value (from the output validator), not a list.